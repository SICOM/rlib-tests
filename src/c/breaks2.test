COMMAND=./breaks2
FORMATS="txt"
EXPECTED_pdf=
EXPECTED_xml=
EXPECTED_txt=
EXPECTED_csv=
EXPECTED_html=
