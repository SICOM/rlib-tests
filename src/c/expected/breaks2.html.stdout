<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
 "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<style type="text/css">
pre { margin:0; padding:0; margin-top:0; margin-bottom:0; font-size:12pt;}
body { background-color: #ffffff;}
TABLE { border: 0; border-spacing: 0; padding: 0; width:100%; }
</style>
<title>RLIB Report</title></head>
<body>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="62" style="font-size: 13px; font-weight: bold;">Breaks Example To Demonstrate Mixed Expressions With Variables</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:4px; background-color:#ffffff; width:100%"></td></tr></table>
<table><!--start from part table--><tr><!--start from part tr--><td><!--started from part td--><!--start pages across--><div><pre style="font-size: 1pt"><span data-col="1" data-width="7" style="font-size: 11px; ">Company</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="2" data-width="8" style="font-size: 11px; ">District</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="3" data-width="6" style="font-size: 11px; ">Region</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="4" data-width="5" style="font-size: 11px; ">Store</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="10" style="font-size: 11px; ">     Sales</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; "> District share</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; "> Regional share</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">  Company share</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:1px; background-color:#000000; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:4px; background-color:#ffffff; width:100%"></td></tr></table>
<div><pre style="font-size: 1pt"><span data-col="1" data-width="24" style="font-size: 11px; ">Foo, Inc.               </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="15" style="font-size: 11px; ">      $17200.00</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#000000; width:100%"></td></tr></table>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="2" data-width="23" style="font-size: 11px; ">Region A               </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="15" style="font-size: 11px; ">       $6600.00</span><span data-col="0" data-width="33" style="font-size: 11px; ">&nbsp;                                </span><span data-col="0" data-width="15" style="font-size: 11px; ">          38.37</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#000000; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:3px; background-color:#ffffff; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#ffffff; width:100%"></td></tr></table>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="2" style="font-size: 11px; ">&nbsp; </span><span data-col="3" data-width="22" style="font-size: 11px; ">District X            </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="15" style="font-size: 11px; ">       $2800.00</span><span data-col="0" data-width="17" style="font-size: 11px; ">&nbsp;                </span><span data-col="0" data-width="15" style="font-size: 11px; ">          42.42</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          16.28</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:1px; background-color:#000000; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#ffffff; width:100%"></td></tr></table>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="3" style="font-size: 11px; ">&nbsp;  </span><span data-col="4" data-width="26" style="font-size: 11px; ">Store #1                  </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="10" style="font-size: 11px; ">  $1500.00</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          53.57</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          22.73</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">           8.72</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="8" style="font-size: 11px; ">  1 /  4</span></pre></div>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="3" style="font-size: 11px; ">&nbsp;  </span><span data-col="4" data-width="26" style="font-size: 11px; ">Store #2                  </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="10" style="font-size: 11px; ">  $1300.00</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          46.43</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          19.70</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">           7.56</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="8" style="font-size: 11px; ">  2 /  4</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#ffffff; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:1px; background-color:#000000; width:100%"></td></tr></table>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="2" style="font-size: 11px; ">&nbsp; </span><span data-col="3" data-width="22" style="font-size: 11px; ">District X            </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="15" style="font-size: 11px; ">       $2800.00</span><span data-col="0" data-width="17" style="font-size: 11px; ">&nbsp;                </span><span data-col="0" data-width="15" style="font-size: 11px; ">          42.42</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          16.28</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:1px; background-color:#000000; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#ffffff; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#ffffff; width:100%"></td></tr></table>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="2" style="font-size: 11px; ">&nbsp; </span><span data-col="3" data-width="22" style="font-size: 11px; ">District Y            </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="15" style="font-size: 11px; ">       $3800.00</span><span data-col="0" data-width="17" style="font-size: 11px; ">&nbsp;                </span><span data-col="0" data-width="15" style="font-size: 11px; ">          57.58</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          22.09</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:1px; background-color:#000000; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#ffffff; width:100%"></td></tr></table>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="3" style="font-size: 11px; ">&nbsp;  </span><span data-col="4" data-width="26" style="font-size: 11px; ">Store #3                  </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="10" style="font-size: 11px; ">  $2000.00</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          52.63</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          30.30</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          11.63</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="8" style="font-size: 11px; ">  3 /  4</span></pre></div>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="3" style="font-size: 11px; ">&nbsp;  </span><span data-col="4" data-width="26" style="font-size: 11px; ">Store #4                  </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="10" style="font-size: 11px; ">  $1800.00</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          47.37</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          27.27</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          10.47</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="8" style="font-size: 11px; ">  4 /  4</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#ffffff; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:1px; background-color:#000000; width:100%"></td></tr></table>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="2" style="font-size: 11px; ">&nbsp; </span><span data-col="3" data-width="22" style="font-size: 11px; ">District Y            </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="15" style="font-size: 11px; ">       $3800.00</span><span data-col="0" data-width="17" style="font-size: 11px; ">&nbsp;                </span><span data-col="0" data-width="15" style="font-size: 11px; ">          57.58</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          22.09</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:1px; background-color:#000000; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#ffffff; width:100%"></td></tr></table>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="2" data-width="23" style="font-size: 11px; ">Region A               </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="15" style="font-size: 11px; ">       $6600.00</span><span data-col="0" data-width="33" style="font-size: 11px; ">&nbsp;                                </span><span data-col="0" data-width="15" style="font-size: 11px; ">          38.37</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#000000; width:100%"></td></tr></table>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="2" data-width="23" style="font-size: 11px; ">Region B               </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="15" style="font-size: 11px; ">      $10600.00</span><span data-col="0" data-width="33" style="font-size: 11px; ">&nbsp;                                </span><span data-col="0" data-width="15" style="font-size: 11px; ">          61.63</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#000000; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:3px; background-color:#ffffff; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#ffffff; width:100%"></td></tr></table>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="2" style="font-size: 11px; ">&nbsp; </span><span data-col="3" data-width="22" style="font-size: 11px; ">District M            </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="15" style="font-size: 11px; ">       $4800.00</span><span data-col="0" data-width="17" style="font-size: 11px; ">&nbsp;                </span><span data-col="0" data-width="15" style="font-size: 11px; ">          45.28</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          27.91</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:1px; background-color:#000000; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#ffffff; width:100%"></td></tr></table>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="3" style="font-size: 11px; ">&nbsp;  </span><span data-col="4" data-width="26" style="font-size: 11px; ">Store #11                 </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="10" style="font-size: 11px; ">  $2500.00</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          52.08</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          23.58</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          14.53</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="8" style="font-size: 11px; ">  1 /  4</span></pre></div>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="3" style="font-size: 11px; ">&nbsp;  </span><span data-col="4" data-width="26" style="font-size: 11px; ">Store #12                 </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="10" style="font-size: 11px; ">  $2300.00</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          47.92</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          21.70</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          13.37</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="8" style="font-size: 11px; ">  2 /  4</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#ffffff; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:1px; background-color:#000000; width:100%"></td></tr></table>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="2" style="font-size: 11px; ">&nbsp; </span><span data-col="3" data-width="22" style="font-size: 11px; ">District M            </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="15" style="font-size: 11px; ">       $4800.00</span><span data-col="0" data-width="17" style="font-size: 11px; ">&nbsp;                </span><span data-col="0" data-width="15" style="font-size: 11px; ">          45.28</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          27.91</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:1px; background-color:#000000; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#ffffff; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#ffffff; width:100%"></td></tr></table>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="2" style="font-size: 11px; ">&nbsp; </span><span data-col="3" data-width="22" style="font-size: 11px; ">District N            </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="15" style="font-size: 11px; ">       $5800.00</span><span data-col="0" data-width="17" style="font-size: 11px; ">&nbsp;                </span><span data-col="0" data-width="15" style="font-size: 11px; ">          54.72</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          33.72</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:1px; background-color:#000000; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#ffffff; width:100%"></td></tr></table>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="3" style="font-size: 11px; ">&nbsp;  </span><span data-col="4" data-width="26" style="font-size: 11px; ">Store #13                 </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="10" style="font-size: 11px; ">  $3000.00</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          51.72</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          28.30</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          17.44</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="8" style="font-size: 11px; ">  3 /  4</span></pre></div>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="3" style="font-size: 11px; ">&nbsp;  </span><span data-col="4" data-width="26" style="font-size: 11px; ">Store #14                 </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="10" style="font-size: 11px; ">  $2800.00</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          48.28</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          26.42</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          16.28</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="8" style="font-size: 11px; ">  4 /  4</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#ffffff; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:1px; background-color:#000000; width:100%"></td></tr></table>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="2" style="font-size: 11px; ">&nbsp; </span><span data-col="3" data-width="22" style="font-size: 11px; ">District N            </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="15" style="font-size: 11px; ">       $5800.00</span><span data-col="0" data-width="17" style="font-size: 11px; ">&nbsp;                </span><span data-col="0" data-width="15" style="font-size: 11px; ">          54.72</span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="0" data-width="15" style="font-size: 11px; ">          33.72</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:1px; background-color:#000000; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#ffffff; width:100%"></td></tr></table>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="2" data-width="23" style="font-size: 11px; ">Region B               </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="15" style="font-size: 11px; ">      $10600.00</span><span data-col="0" data-width="33" style="font-size: 11px; ">&nbsp;                                </span><span data-col="0" data-width="15" style="font-size: 11px; ">          61.63</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#ffffff; width:100%"></td></tr></table>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#000000; width:100%"></td></tr></table>
<div><pre style="font-size: 1pt"><span data-col="1" data-width="24" style="font-size: 11px; ">Foo, Inc.               </span><span data-col="0" data-width="1" style="font-size: 11px; ">&nbsp;</span><span data-col="5" data-width="15" style="font-size: 11px; ">      $17200.00</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:2px; background-color:#ffffff; width:100%"></td></tr></table>
</td><!--ended from part td--><!--end pages across--></tr><!--ended from part tr--></table><!--ended from part table--></body></html>
